Support Jest ^28

```
npm ERR! Could not resolve dependency:
npm ERR! peer jest@"^24.0.0 || ^25.0.0 || ^26.0.0 || ^27.0.0" from jest-mock-extended@2.0.5
npm ERR! node_modules/jest-mock-extended
npm ERR!   jest-mock-extended@"^2.0.5" from @configs/jest@1.0.0
```

Should be as easy as updating this line https://github.com/marchaos/jest-mock-extended/blob/master/package.json#L35

--- a/package.json
+++ b/package.json
@@ -23,16 +23,16 @@
     "ts-essentials": "^7.0.3"
   },
   "devDependencies": {
-    "@types/jest": "^26.0.24",
+    "@types/jest": "^27.4.1",
     "coveralls": "^3.1.1",
-    "jest": "^26.6.3",
+    "jest": "^28.0.2",
     "prettier": "^2.3.2",
     "rimraf": "^3.0.2",
-    "ts-jest": "^26.5.6",
+    "ts-jest": "^28.0.0-next.2",
     "typescript": "^4.3.5"
   },
   "peerDependencies": {
-    "jest": "^24.0.0 || ^25.0.0 || ^26.0.0 || ^27.0.0",
+    "jest": "^24.0.0 || ^25.0.0 || ^26.0.0 || ^27.0.0 || ^28.0.0",
     "typescript": "^3.0.0 || ^4.0.0"
   },
   "author": "Marc McIntyre <marchaos@gmail.com>",

--- a/package.json
+++ b/package.json
@@ -23,12 +23,12 @@
     "ts-essentials": "^7.0.3"
   },
   "devDependencies": {
-    "@types/jest": "^27.4.1",
+    "@types/jest": "^27.5.0",
     "coveralls": "^3.1.1",
-    "jest": "^28.0.2",
+    "jest": "^28.1.0",
     "prettier": "^2.3.2",
     "rimraf": "^3.0.2",
-    "ts-jest": "^28.0.0-next.2",
+    "ts-jest": "^28.0.1",
     "typescript": "^4.3.5"
   },
   "peerDependencies": {

